compiler/src/iree/compiler/Codegen/LLVMCPU/DispatchABI.h

```diff
@@ -267,9 +267,11 @@ class HALDispatchABI {
   // Loads the processor ID the code is (most likely) being run on.
   // Equivalent to:
   //   uint32_t processor_id = state->processor_id;
+  Type getProcessorIDType();
```

Comment:
the commends were on the previous methods so you'll want to move these type accessors around them instead of interleaving